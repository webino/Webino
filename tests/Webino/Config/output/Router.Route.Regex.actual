array(
	'options' => array('route' => '<example>/.*', 'spec' => '%example%'),
	'type' => 'regex',
)
