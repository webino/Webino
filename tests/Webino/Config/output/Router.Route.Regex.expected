array(
	'type' => 'regex',
	'options' => array('regex' => '<example>/.*', 'spec' => '%example%'),
)
